tests/lib/rules/jsx-curly-brace-presence.js

```diff
@@ -32,6 +32,10 @@ ruleTester.run('jsx-curly-brace-presence', rule, {
     {
       code: '<App {...props}>foo</App>'
     },
+    {
+      code: '<>foo</>',
```

Comment:
This rule doesn't really care whether it's an element or a fragment, so I just added one simple valid test, and one invalid.